src/index.js

```diff
@@ -1,12 +1,17 @@
 const { router, text } = require('bottender/router');
 const { PythonShell } = require('python-shell')
+const mysql = require('mysql');
 
 async function SayHi(context) {
   await context.sendText('Hi!');
 }
 
+async function GetUserID(event) {
+  await event.source.userId;
+}
+
 async function ParseUrl(context) {
-  const query = context.event.text;
+  let query = context.event.text;
   context.setState({
     url: query.includes(':') ? query.split(':')[1] : query.split('：')[1],
```

Comment:
As we found yesterday, if input url contains “https://“, this line will not do well.
Also, we need to add a http or https prefix if user doesn’t provide it, for the usage of crawler in Python.